lib/routes/ehentai/ehapi.js

```diff
@@ -0,0 +1,146 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const config = require('@/config').value.ehentai;
+
+const headers = {};
+const has_cookie = config.ipb_member_id && config.ipb_pass_hash && config.sk;
+if (has_cookie) {
+    const ipb_member_id = config.ipb_member_id;
+    const ipb_pass_hash = config.ipb_pass_hash;
+    const sk = config.sk;
+    headers.cookie = `ipb_member_id=${ipb_member_id};ipb_pass_hash=${ipb_pass_hash};sk=${sk}`;
+}
+
+function generalGot(url) {
+    return got({
+        method: 'get',
+        url: url,
+        headers: headers,
+    });
+}
```

Comment:
Isn't the `got.get` not good enough? Why wrap it in a separate function?

Reply:
You right, it is good enough, just use it. Related code modified.